agent/consul/state/config_entry_test.go

```diff
@@ -2065,6 +2067,53 @@ func TestStore_ReadDiscoveryChainConfigEntries_SubsetSplit(t *testing.T) {
 	require.Len(t, entrySet.Services, 1)
 }
 
+func TestStore_ReadDiscoveryChainConfigEntries_PeerLocality(t *testing.T) {
```

Comment:
Nit. There's not really anything related to locality in this test, so the name is a little confusing for outsiders who are unfamiliar with this PR.